test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -1011,16 +1023,44 @@ func isAutoscalerEnabled(expectedMaxNodeCountInTargetPool int) (bool, error) {
 	return false, nil
 }
 
+func getClusterLocation() string {
+	if isRegionalCluster() {
+		return "--region=" + framework.TestContext.CloudConfig.Region
+	} else {
+		return "--zone" + framework.TestContext.CloudConfig.Zone
+	}
+}
+
+func getGcloudCommand(commandTrack string, args []string) []string {
+	command := []string{"gcloud"}
+	if commandTrack == "beta" || commandTrack == "alpha" {
+		command = append(command, commandTrack)
+	}
+	command = append(command, args...)
+	return command
+}
+
+func isRegionalCluster() bool {
+	// TODO(bskiba): Use an appropriate indicator that the cluster is regional.
+	return framework.TestContext.CloudConfig.MultiZone
+}
+
 func enableAutoscaler(nodePool string, minCount, maxCount int) error {
 	glog.Infof("Using gcloud to enable autoscaling for pool %s", nodePool)
 
-	output, err := execCmd("gcloud", "container", "clusters", "update", framework.TestContext.CloudConfig.Cluster,
+	location := getClusterLocation()
```

Comment:
Can we maybe move adding project and location flags to getGcloudCommand func? This would both make that function more useful and reduce the repetitive code everywhere else

Reply:
Good idea. Done.